Makefile

```diff
@@ -42,7 +42,7 @@ http-docs:
 
 .PHONY: test
 test:
-	set -o pipefail && $(GOTEST) | sed ''/PASS/s//$$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$$(printf "\033[31mFAIL\033[0m")/'' | grep -v RUN
+	set pipefail -o && $(GOTEST) | sed ''/PASS/s//$$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$$(printf "\033[31mFAIL\033[0m")/'' | grep -v RUN
```

Comment:
I got 3 results on google for "set pipefail -o" and over 100k for "set -o pipefail"... I'm no `bash` expert but I believe this works because the command is wrong and probably ignored :thinking: 

Looking at [GNU](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html), the correct usage is as it was before:

- set [--abefhkmnptuvxBCEHPT] [-o option-name] [argument …]
- set [+abefhkmnptuvxBCEHPT] [+o option-name] [argument …]

Also, `pipefail` is an option for `-o` (also at [GNU](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)):

- -o option-name
- Set the option corresponding to option-name: [ ..., pipefail ,....]

Here's an [MIT](https://sipb.mit.edu/doc/safe-shell/) guide to safe shell scripting, might help too. I suggest we try adding `bash` as a persistent dependency in the dockerfile and running make test from inside it's shell.